### PR TITLE
reagents value nerf and standardization.

### LINKED ISSUE #10294: Cargo Balance Issue
## Reproduction:

Cigarettes give you way too much money and you can buy restocking kits for the vendor. Keep restocking the vendor, sell the cigs, and you make an obscene amount of money. Service borgs can print cigs and you get the same effect. 

This was like, less than 10 vendor kits, which only "cost" 800 each. Even at 10 kits. that's only 8000 credits for over 100k in cigs.

image

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Introducing defines for reagents velues (because the values were unconsistent as fuck but it'd be pretty impossible to procedurally generate the prices).

Tweaking the values all around with the main intention of making most commonplace, not harmful reagents be worth at most 1 cr per unit, and harder, unique, unsynthetizable and game ender ones up to 10, 30 and 300 cr/u.

I mostly won't care if some of you might think this is excessive (which is not). I have plans of introducing personal cargo exports to allow productive players to get a tiny share (depending on typology of export) of credits from what they sell through cargo once economy is good to go, and I got no intention for it to be busted by the currently huge unbalanced status of this feature.

## Why It's Good For The Game
Balance and nerf. Also these constant defines are way easier to toy with than magic numbers.
This will close #10294

## Changelog
:cl:
balance: nerfed reagents cargo values.
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
I agree fully with this.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -142,6 +142,7 @@
 	description = "A synthetically produced imitation of blood."
 	taste_description = "oily"
```
Comment (Putnam3145) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: i know it's not in the PR title/desc but can you change this to "oil" while you're at it cause "you taste oily" has been grating on me and i am lazy

File: modular_citadel/code/modules/reagents/chemistry/reagents/SDGF.dm [hunk 2]
```diff
@@ -52,6 +52,7 @@ IMPORTANT FACTORS TO CONSIDER WHILE BALANCING
 	inverse_chem_val 		= 0.5
 	inverse_chem		= /datum/reagent/impure/SDZF
 	can_synth = TRUE
+	value = REAGENT_VALUE_EXCEPTIONAL
```
Comment (Putnam3145) [inline modular_citadel/code/modules/reagents/chemistry/reagents/SDGF.dm]: while it does require uranium, i can still mass produce this stuff pretty quickly?

File: modular_citadel/code/modules/reagents/chemistry/reagents/healing.dm [hunk 3]
```diff
@@ -100,6 +101,7 @@
 	data = list("grown_volume" = 0, "injected_vol" = 0)
 	var/borrowed_health
 	color = "#FFDADA"
+	value = REAGENT_VALUE_VERY_RARE
```
Comment (Putnam3145) [inline modular_citadel/code/modules/reagents/chemistry/reagents/healing.dm]: "takes a long time" but making it is embarrassingly parallel and cheap--1u of synthflesh becomes 5u of synthtissue

File: modular_citadel/code/modules/reagents/chemistry/reagents/SDGF.dm (hunk 2 above)
Comment (Ghommie) [reply modular_citadel/code/modules/reagents/chemistry/reagents/SDGF.dm]: I didn't know, thanks for pointing out.

**Comment by davidchan13** [comment]
Space Lube worth 5 times more per unit than Omnizine?
Ammonia worth 10 credits per unit?

I hope I'm not reading these right.

**Comment by Ghommie** [comment]
@davidchan13 You are reading it wrong, expand the files and try again.

**Pull request merged**